src/test/consensus/Consensus_test.cpp

```diff
@@ -214,42 +213,8 @@ class Consensus_test : public beast::unit_test::suite
                 auto const& lgrID = p.prevLedgerID();
                 BEAST_EXPECT(lgrID.seq == 1);
 
-                // If peer 0 is participating
-                if (isParticipant)
-                {
-                    BEAST_EXPECT(p.prevProposers() == sim.peers.size() - 1);
-                    // Due to the network link delay settings
-                    //    Peer 0 initially proposes {0}
-                    //    Peer 1 initially proposes {1}
-                    //    Peers 2-5 initially propose {2,3,4,5}
-                    // Since peers 2-5 agree, 4/6 > the initial 50% threshold
-                    // on disputed transactions, so Peer 0 and 1 change their
-                    // position to match peers 2-5 and declare consensus now that
-                    // 5/6 proposed positions match (themselves and peers 2-5).
-                    //
-                    // Peers 2-5 do not change position, since tx 0 or tx 1 have
-                    // less than the 50% initial threshold.  They also cannot
-                    // declare consensus, since 4/6 < 80% threshold agreement on
-                    // current positions.  Instead, they have to wait an
-                    // additional timerEntry call for the updated peer 0 and
-                    // peer 1 positions to arrive.  Once they do, now peers 2-5
-                    // see complete agreement and declare consensus
-                    if (p.id > 1)
-                        BEAST_EXPECT(
-                            p.prevRoundTime() > sim.peers[0].prevRoundTime());
-                }
-                else  // peer 0 is not participating
-                {
-                    auto const proposers = p.prevProposers();
-                    if (p.id <= 1)
-                        BEAST_EXPECT(proposers == sim.peers.size() - 2);
-                    else
-                        BEAST_EXPECT(proposers == sim.peers.size() - 3);
-
-                    // so all peers should have closed together
-                    BEAST_EXPECT(
-                        p.prevRoundTime() == sim.peers[0].prevRoundTime());
-                }
+                BEAST_EXPECT(p.prevProposers() == sim.peers.size() - 1);
+                BEAST_EXPECT(p.prevRoundTime() == sim.peers[0].prevRoundTime());
```

Comment:
Yes, that is the only change for now.  The nodes still reach consensus, it just takes peers 1-4 longer.

As part of building out the framework, there is a now a better model for writing an explicit checker that would verify the number of proposals each node sent.  I'll target this spot for an upgrade when it gets released.